Src/WitsmlExplorer.Frontend/components/ContentViews/table/ContentTable.tsx

```diff
@@ -180,7 +180,10 @@ export const ContentTable = (contentTableProps: ContentTableProps): React.ReactE
                 <Fragment key={row.id}>
                   <StyledTr
                     selected={row.getIsSelected()}
-                    onContextMenu={(e) => onRowContextMenu(e, row)}
+                    onContextMenu={async (e) => {
+                      await row.toggleSelected(true);
```

Comment:
I am getting a warning "'await' has no effect on the type of this expression.ts(80007)", but removing `await` make the context menu behave as if nothing is selected, so a comment would be nice to ensure that the `await` is not accidentally removed.
```suggestion
                      // await selection to ensure that the context menu detects that a row has been selected
                      await row.toggleSelected(true);
```

Reply:
Ok, I put the comment there